pkgs/development/ocaml-modules/ninja_utils/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, fetchurl, buildDunePackage, re }:
+
+buildDunePackage rec {
+  pname = "ninja_utils";
+  version = "0.9.0";
+
+  src = fetchurl {
+    url = "https://github.com/CatalaLang/ninja_utils/archive/refs/tags/${version}.tar.gz";
```

Comment:
Please use `fetchzip` (directly or through `fetchFromGitHub`) for this kind of URLs.